deploy/charts/istio-operator/templates/operator-statefulset.yaml

```diff
@@ -69,6 +69,12 @@ spec:
           {{- end }}
           - "--leader-election-namespace={{ .Values.leaderElection.namespace }}"
           {{- end }}
+          {{- if and (.Values.webhook.enabled) (eq .Values.istioVersion "1.7") }}
```

Comment:
Could this be something like "greater than" (if that works with strings) instead of "equals" to support newer versions as well out of the box in the future? (Istio 1.8, etc.)